snippets/number-to-array-of-digits.md

```diff
@@ -0,0 +1,17 @@
+### Number to array of digits
+
+use `parseInt()` to get only the integer part of the quotient.
+use `Array.reverse()` to return the array in the same order as the number. 
+
+```js
+const num2array = (n) =>{
+	let arr = [];
+	while (n>0) { 
+		arr.push(n%10); 
+		n = parseInt(n/10); 
+	}
+	return arr.reverse();
+}
+
```

Comment:
You can use `(''+n).split('').map(parseInt)` to achieve exactly the same thing. Probably more performant, too.

Reply:
done